sdk/nextjs/src/client/useIdentifyUser.ts

```diff
@@ -0,0 +1,31 @@
+'use client'
+import { DevCycleUser } from '@devcycle/js-client-sdk'
+import { updateDVCCookie } from './updateDVCCookie'
+import { useRouter } from 'next/navigation'
+import { startTransition, useContext } from 'react'
+import { DevCycleClientContext } from './DevCycleClientsideProvider'
+
+/**
+ * Hook that returns an identify function you can use to identify a user clientside.
+ * Calling this will trigger a `router.refresh()` which allows server components to re-render with the new identity
+ */
+export const useIdentifyUser = (): ((user: DevCycleUser) => void) => {
+    const context = useContext(DevCycleClientContext)
+    const router = useRouter()
+
+    if (!context.enableClientsideIdentify) {
+        console.warn(
```

Comment:
should we just throw instead of doing nothing?

Reply:
we could, I just thought it might be an annoying pitfall to suddenly start throwing from peoples' code if that option is changed or something

Reply:
it could also be a pitfall that they expect this to do something while turning this option on, but I'll leave that judgment up to you